<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>QPlainTextEdit Class</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>ReadOnly Disable</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="selectAll">
       <property name="text">
        <string>selectAll</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Copy">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cut">
       <property name="text">
        <string>Cut</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Paste">
       <property name="text">
        <string>Paste</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="blockCountA">
       <property name="text">
        <string>+Block Count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="blockCountE">
       <property name="text">
        <string>-Block Count</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="Append">
     <property name="text">
      <string>Append</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Undo">
     <property name="text">
      <string>Undo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Redo">
     <property name="text">
      <string>Redo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="OverWriteMode">
     <property name="text">
      <string>OverWriteMode Enable</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
